pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -285,6 +273,9 @@ func getListenersByLoadBalancerID(client *gophercloud.ServiceClient, id string)
 		return true, nil
 	})
 	if err != nil {
+		if isNotFound(err) {
```

Comment:
Yeah, I know it, when len(...) is 0, K8S will return  ErrNotFound and caller can't use isNotFound(...). So I think the better way is that calling k8s function with ErrNotFound and calling OpenStack function with isNotFound(...). Do you think so?